modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/portlet/action/EditCommerceOrderMVCActionCommand.java

```diff
@@ -130,6 +133,9 @@ else if (cmd.equals("requestedDeliveryDate")) {
 			else if (cmd.equals("reorder")) {
 				_reorderCommerceOrder(actionRequest);
 			}
+			else if (cmd.equals("requestedQuote")) {
```

Comment:
I think the command verb should be "at present" (like reorder, selectBil***)

Reply:
@luca-pellizzon , same for this one. If I change the value for this, I would also have to change the corresponding method name to `_updateRequestQuoteCommerceOrder`. cc/ @smottal

Reply:
I would go with requestQuote like we have "update" or "add" as other actions.
Regarding the method name, I would go with a simple _requestQuote() without the "update", so that it is more similar to "addBillingAddress" ecc